src/main/resources/com/google/api/codegen/php/standalone_sample.snip

```diff
@@ -326,3 +328,71 @@
         $unhandledCase: {@initValue.type}$
     @end
 @end
+
+@private processResponse(sample)
+    @if sample.outputs.size == 0
+        printf($response . PHP_EOL);
```

Comment:
I think printing proto object should definitely be supported. From reading `print_r` manual, it's possible to pass `$return` argument so that it returns a string. In which case is it possible to do something like this?

```php
printf("foo %s bar", print_r($obj, $return=true));
```
(I don't actually know how PHP syntax works)

Reply:
Here's my proposal:

 1. For this PR, I'm alright if trying to print a protobuf message object blows up with a **runtime error** 👍
 2. As @michaelbausor said, let's definitely make sure this is on the protobuf teams' radar to fix!
 3. As a follow-up to this PR, to fix the runtime error, let's render PHP that prints *something* about the object *without blowing up* -- let's roll with what you put, @pongad, using `print_r($obj, $return=true)`

/cc @tmatsuo @bshaffer – thoughts?

Reply:
Yes, and that would be a sensible way to do it I think. Syntax would be:
```php
printf("foo %s bar", print_r($obj, true));
```

(I have just been looking at other options, and can't find anything better. The output of var_dump is much nicer that print_r, but can't be captured in a string. var_export looked promising, but fails because it cannot handle circular references (which all protobuf objects contain in their internals)).

Reply:
+1 to proposal above from @beccasaurus